src/frontend/get-jira-configuration.ts

```diff
@@ -19,60 +19,58 @@ function mapSyncStatus(syncStatus: SyncStatus = SyncStatus.PENDING): string {
 	}
 }
 
-export async function getInstallation(client, subscription: SubscriptionClass, reqLog?: Logger) {
+export interface InstallationResults {
```

Comment:
Should we use type instead of interface?

Reply:
Why?  Type seems like more work to implement?

Reply:
For consistency, because everywhere else we use types for data objects

```
export type InstallationResults = {
	fulfilled: AppInstallation[],
	rejected: FailedAppInstallation[],
	total: number
}
```

Seems that's all we need to do (If I am not missing something)

Reply:
I see 38 matches for `interface` use and 35 for `type` use in the codebase.  It's definitely not the standard.

I normally prefer `interface` for objects as it's easier to read when you're doing `extends` or using utils like `Partial`, and you can't used  `implements` with types.  For example:

```
type Foo = { blah: string; };
type Bar = { weee:boolean };
type FooBarType = foo & bar & { something: string; };

interface FooBar extends Foo, Bar {
   something: string;
}
```

Personally, I find interface to be cleaner and there's some nifty tricks you can do with it too.  Types are good for unions and merges of types.